.github/workflows/ruby.yml

```
@@ -0,0 +1,58 @@
+name: dc-ruby-devtest
+
+on:
+  push:
+    branches: '*'
+  pull_request:
+    branches:  '*'
+
+jobs:
+  pipeline:
+    runs-on: ubuntu-latest
+    env:
+      RAILS_ENV: test
+      DATABASE_USER: user
+      DATABASE_PASSWORD: password
+      DATABASE_HOST: localhost
+      DATABASE_URL: postgresql://user:password@localhost/delivery_ruby_test
+
+    services:
+      db:
+        image: postgres
+        ports: ['5432:5432']
+        env:
+          POSTGRES_DB: delivery_ruby_test
+          POSTGRES_USER: user
+          POSTGRES_PASSWORD: password
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Setup Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: '2.7.1'
+
+    - name: Run Install
+      run: |
+        date
+        sudo apt-get -yqq install libpq-dev
+        gem update --system
+        gem install rake -v 13.0.1
+        gem install bundler -v "~>2.0"
+        bundle install
+
+    - name: Run Linter (rubocop)
+      run: |
+        bundle exec rubocop --format json -o report.json
+
+    - name: Run Tests (rspec)
+      run: |
+        bundle exec rails db:create db:migrate
+        bundle exec rspec --fail-fast
```

I believe we don't need the fail fast here, It's better to have the full result here and let the fail fast to development env only

---

But wouldn't this consume too many CI time? 💸

---

If the cost is not a concern I can remove. What do you think?

---

Well, I think it wouldn't be a problem 🤔 Your call here

---

Idk too... @dminella you that is more familiar with our github budget, how should we proceed? 🤔

---

I'll keep all suit running for now, if someone complains we change it.